src/app/icd/server/ICDConfigurationData.h

```diff
@@ -92,7 +96,22 @@ class ICDConfigurationData
 
 private:
     // Singleton Object
-    ICDConfigurationData() = default;
+    ICDConfigurationData()
+    {
+        // Initialize feature map
+#if CHIP_CONFIG_ENABLE_ICD_LIT
+        mFeatureMap.Set(app::Clusters::IcdManagement::Feature::kLongIdleTimeSupport);
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+        mFeatureMap.Set(app::Clusters::IcdManagement::Feature::kCheckInProtocolSupport);
+#endif // CHIP_CONFIG_ENABLE_ICD_CIP
+#if CHIP_CONFIG_ENABLE_ICD_UAT
+        mFeatureMap.Set(app::Clusters::IcdManagement::Feature::kUserActiveModeTrigger);
+#endif // CHIP_CONFIG_ENABLE_ICD_UAT
+#if CHIP_CONFIG_ENABLE_ICD_DSLS
```

Comment:
This one, on the other hand, _is_ only allowed when CHIP_CONFIG_ENABLE_ICD_LIT, no?

We should consider just having a static_assert that we don't have CHIP_CONFIG_ENABLE_ICD_DSLS when CHIP_CONFIG_ENABLE_ICD_LIT is false.

Reply:
Yes, DSLS can be enabled only when LITS is supported. Added the assert to the BUILD.gn